library/sources/http-server/shouldDiscoverRoute.ts

```diff
@@ -34,23 +34,27 @@ export function shouldDiscoverRoute({
     return false;
   }
 
-  return segments.every(isAllowedExtension);
+  // Check for every file segment if it contains an file extension and if it should be discovered or ignored
+  return segments.every(shouldDiscoverExtension);
```

Comment:
We only need to do this for the full string right? Not for every segment, maybe we can refactor?

Reply:
Required for `/index.php/something`